app/jobs/scheduled/clean_up_uploads.rb

```diff
@@ -1,25 +1,36 @@
 module Jobs
-
   class CleanUpUploads < Jobs::Scheduled
     every 1.hour
 
     def execute(args)
       return unless SiteSetting.clean_up_uploads?
 
+      ignore_urls = []
```

Comment:
Ah good point. We will probably need to do that. I'll have a look again next Monday